sdk/identity/Azure.Identity/src/EnvironmentCredential.cs

```diff
@@ -18,7 +18,7 @@ namespace Azure.Identity
     /// <item><term>AZURE_TENANT_ID</term><description>The Azure Active Directory tenant(directory) ID.</description></item>
     /// <item><term>AZURE_CLIENT_ID</term><description>The client(application) ID of an App Registration in the tenant.</description></item>
     /// <item><term>AZURE_CLIENT_SECRET</term><description>A client secret that was generated for the App Registration.</description></item>
-    /// <item><term>AZURE_CLIENT_CERTIFICATE_LOCATION</term><description>A path to the certificate that was generate for the App Registration.</description></item>
+    /// <item><term>AZURE_CLIENT_CERTIFICATE_PATH</term><description>A path to the certificate that was generate for the App Registration.</description></item>
```

Comment:
Since you're fixing this line anyway we should probably reword this a bit to make it more correct.

> A path to certificate and private key pair in PEM or PFX format, which can authenticate the App Registration.

Reply:
Done